src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
   [RS0026](https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md#rs0026-do-not-add-multiple-public-overloads-with-optional-parameters).
   ([#1218](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1218))
 
+* Update `MetricEtwDataTransport` which is  the `EventSource` implementation for
```

Comment:
I guess the user would be very confused by this message since they are not supposed to understand the internals (e.g. MetricEtwDataTransport might mean nothing to them).

Do we really need a changelog entry?

Reply:
I added this CHANGELOG entry so that we can easily figure out which version got this fix.

Update the wording.

Reply:
> so that we can easily figure out which version got this fix.

I guess we want to look at the commit history (e.g. `git log | grep ...`) rather than CHANGELOG (which is for the users, not the maintainers/owners)?

Reply:
I think it's much quicker to first look for any noteworthy changes for a given release in the CHANGELOG, than going through git log which is the comprehensive source of truth.